src/lib/dev-environment/dev-environment-core.js

```diff
@@ -440,20 +439,16 @@ export async function importMediaPath( slug: string, filePath: string ) {
  *   - If there is a newer version of the WordPress version currently used
  *   - A choice to use a different image
  *
- * @param  {Object=} instancePath Path to local profile
+ * @param  {Object=} slug slug
  * @param  {Object=} options options
  */
-async function updateWordPressImage( instancePath, options ) {
+async function updateWordPressImage( slug, options ) {
 	const versions = await getVersionList();
 	const refRgx = new RegExp( /\d+\.\d+(?:\.\d+)?/ );
-	const vsnRgx = new RegExp( /\$wp_version \= \'(.*)\'\;/ );
-	const landoFile = `${ instancePath }/.lando.yml`;
-	const versionFile = `${ instancePath }/${ DEV_ENVIRONMENT_WORDPRESS_VERSION_FILE }`;
+	const instancePath = getEnvironmentPath( slug );
 
-	// If versionFile doesn't exist it means that the environment has not been initiated.
-	if ( ! fs.existsSync( versionFile ) ) {
-		return;
-	}
+	// Get the current image tag that the WP image is currently using in the .lando.yml file
+	const current = readEnvironmentData( slug );
```

Comment:
This can throw an exception if the env is build with older `vip` version ... we should probably catch it here

Or maybe change the method signature to catch inside it and return null.

Check `update` logic on how it is done there (and potentially change there too) - https://github.com/Automattic/vip/blob/801ed99d178eaacc5639fef7ac41a1e55df9f7bc/src/bin/vip-dev-env-update.js#L80